internal/backend/svc/svc.go

```diff
@@ -127,7 +126,7 @@ func makeFxOpts(cfg *Config, opts RunOptions) []fx.Option {
 		Component("auth", configset.Empty, fx.Provide(authsvc.New)),
 		Component("authjwttokens", authjwttokens.Configs, fx.Provide(authjwttokens.New)),
 		Component("authsessions", authsessions.Configs, fx.Provide(authsessions.New)),
-		Component("authhttmiddleware", authhttpmiddleware.Configs, fx.Provide(authhttpmiddleware.New)),
+		Component("authhttmiddleware", authhttpmiddleware.Configs, fx.Provide(authhttpmiddleware.New), fx.Provide(authhttpmiddleware.AuthorizationHeaderExtractor)),
```

Comment:
It's easy to miss your addition to this line - please use line wrapping like in other `Component` calls.